starlette/schemas.py

```diff
@@ -21,42 +21,66 @@ def render(self, content: typing.Any) -> bytes:
         return yaml.dump(content, default_flow_style=False).encode("utf-8")
 
 
+class Endpoint(typing.NamedTuple):
```

Comment:
I'm a bit wary about calling this `Endpoint`, since the callable is what we're referring to as the "endpoint" elsewhere. Perhaps it should be `EndpointInfo`?

Reply:
yes, makes sense, another alternative could be `PathInfo` as well. I'll stick with `EndpointInfo` for now.